lib/iris/fileformats/netcdf.py

```diff
@@ -2685,28 +2685,47 @@ def add_ellipsoid(ellipsoid):
                     cf_var_grid.false_easting = cs.false_easting
                     cf_var_grid.false_northing = cs.false_northing
 
+                # polar stereo (have to do this before Stereographic because it subclasses it)
+                elif isinstance(cs, iris.coord_systems.PolarStereographic):
+                    if cs.ellipsoid:
+                        add_ellipsoid(cs.ellipsoid)
+                    cf_var_grid.latitude_of_projection_origin = cs.central_lat
+                    cf_var_grid.straight_vertical_longitude_from_pole = (
+                        cs.central_lon
+                    )
+                    cf_var_grid.false_easting = cs.false_easting
+                    cf_var_grid.false_northing = cs.false_northing
+                    # Only one of these should be set
+                    if cs.true_scale_lat is not None:
+                        cf_var_grid.true_scale_lat = cs.true_scale_lat
+                    elif cs.scale_factor_at_projection_origin is not None:
+                        cf_var_grid.scale_factor_at_projection_origin = (
+                            cs.scale_factor_at_projection_origin
+                        )
+                    else:
+                        cf_var_grid.scale_factor_at_projection_origin = (
+                            1.0  # TODO: Is this right?
+                        )
+
                 # stereo
                 elif isinstance(cs, iris.coord_systems.Stereographic):
+                    if cs.ellipsoid:
+                        add_ellipsoid(cs.ellipsoid)
+                    cf_var_grid.longitude_of_projection_origin = cs.central_lon
+                    cf_var_grid.latitude_of_projection_origin = cs.central_lat
+                    cf_var_grid.false_easting = cs.false_easting
+                    cf_var_grid.false_northing = cs.false_northing
+                    # Only one of these should be set
                     if cs.true_scale_lat is not None:
-                        warnings.warn(
-                            "Stereographic coordinate systems with "
-                            "true scale latitude specified are not "
-                            "yet handled"
+                        cf_var_grid.true_scale_lat = cs.true_scale_lat
+                    elif cs.scale_factor_at_projection_origin is not None:
```

Comment:
This is a bit tricky...
According to their CF descriptions, whereas a [PolarStereographic grid-mapping](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#polar-stereographic) can have _either_ a 'standard_parallel'  or 'scale_factor_at_projection_origin' attribute, the plain [Stereographic type](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#_stereographic) _only_ supports the 'scale_factor_at_projection_origin' one.

So, if we get a Stereographic with a standard-parallel, we can either try to _convert_ it to a scale-factor value, or we could just refuse to save it.  
In theory this can't happen with one loaded from CF, so  I think it's probably unlikely.  Rather than take responsibility for a conversion, my choice would be to error it.

Likewise, according to CF, a Stereographic grid-mapping _must_ have a scale factor, and a PolarStereo _must_ have either a scale-factor or a standard-parallel.
In those cases, if the coord-system has neither set, I think it is probably OK and more useful to set a scale-factor of 1.

Reply:
Now raising a `ValueError` as that seems consistent with the nearby code. Could change that though